src/components/Card.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+
+import classes from './Card.module.css';
+import { CardProps } from './Card.types';
+
+const MAX_TEXT_LENGTH = 80;
+
+const Card: FC<CardProps> = ({
+  title,
+  image,
+  text,
+  tags,
+  isDragging,
+  maxTextSize = MAX_TEXT_LENGTH,
+}): React.ReactElement => {
+  const cardTags: string[] = tags?.split(',').slice(0, 2) ?? [];
+
+  return (
+    <div className={`${classes.card} ${isDragging ? classes.dragging : ''}`}>
+      <div className={classes['card-media']}>
+        <img src={image} alt={title}></img>
```

Comment:
Love that you did not forget alt! 🤌🏻
If you have html tag without children, you can for sure use self-closing `<img />`

Reply:
fixed